packages/dependency-extraction-webpack-plugin/package.json

```diff
@@ -20,7 +20,7 @@
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
 	"engines": {
-		"node": ">=14"
+		"node": ">=18"
```

Comment:
I think it essentially means `@wordpress/scripts` needs to bump the minimum required version, too.

Reply:
I bumped that version too, thanks.

The Node version change isn't essential to change, but I don't see much reason to support versions that Node doesn't maintain.